src/kits/tracker/ContainerWindow.cpp

```diff
@@ -562,6 +526,28 @@ BContainerWindow::BContainerWindow(LockingList<BWindow>* list,
 		list->AddItem(this);
 	}
 
+	fRootLayout = new BGroupLayout(B_VERTICAL, 0);
+	fRootLayout->SetInsets(0);
+	SetLayout(fRootLayout);
+	fRootLayout->Owner()->SetViewColor(ui_color(B_PANEL_BACKGROUND_COLOR));
+
+	if (!fIsDeskWindow) {
```

Comment:
This, and the fact that the fIsDeskWindow is needed in the first place means that something is wrong with the view hierarchy. One way to solve this would be to introduce a common base class; I know it's not originally your code, but that's still no reason to build on something broken.

Of course, that could be solved with in a second commit, too.